modules/agar/src/main/ts/ephox/agar/api/Mouse.ts

```diff
@@ -6,6 +6,15 @@ import { Chain } from './Chain';
 import { Step } from './Step';
 import * as UiFinder from './UiFinder';
 
+const click = Clicks.click({});
+const mouseOver = Clicks.mouseOver({});
+const mouseDown = Clicks.mouseDown({});
+const mouseUp = Clicks.mouseUp({});
+const mouseMove = Clicks.mouseMove({});
+const mouseOut = Clicks.mouseOut({});
```

Comment:
Is the style we prefer `({ })`? I see this in most places, maybe we should lint?

```suggestion
const click = Clicks.click({ });
const mouseOver = Clicks.mouseOver({ });
const mouseDown = Clicks.mouseDown({ });
const mouseUp = Clicks.mouseUp({ });
const mouseMove = Clicks.mouseMove({ });
const mouseOut = Clicks.mouseOut({ });
```